calculator.rb

```diff
@@ -0,0 +1,97 @@
+puts "Thank you for choosing my calculator"
+answer = 0
+
+def is_numeric(s)
+  # This begin / rescue block is for exception catching.
+  # We need it because we're trying to convert string to float and it might fail if string contains non-numeric characters
+  # on successful conversion - it will return true, else false
+  begin
+    Float(s)
+  rescue
+    false # not numeric
+  else
+    true # numeric
```

Comment:
Good work figuring out the exception handling here - we'll cover this in class in a couple weeks.

Style nitpick: you should always try to use explicit return statements:

```ruby
begin
  Float(s)
rescue
  return false
else
  return true
end
```